integration-tests/showSolutionListPage.test.js

```diff
@@ -6,6 +6,9 @@ const mocks = () => {
   nock('http://localhost:8080')
     .get('/api/v1/Solutions')
     .reply(200, aSolutionList);
+  nock('http://localhost:8080')
+    .get('/api/v1/Solutions/S1/Public')
+    .reply(200);
 };
 
```

Comment:
I was thinking we would add this to the actual test as this mock is only specific to the click the view solution link test.

Reply:
I only put it in to stop the error - you can change it if you like